scripts/Tools/component_generate_baseline

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+
+"""
+Copies current component history files into baselines
+"""
+
+from standard_script_setup import *
+
+from CIME.case       import Case
+from CIME.hist_utils import generate_baseline
+
+###############################################################################
+def parse_command_line(args, description):
+###############################################################################
+    parser = argparse.ArgumentParser(
+        usage="""\n%s [<casedir>] [--verbose]
+OR
+%s --help
+OR
+%s --test
+
+\033[1mEXAMPLES:\033[0m
+    \033[1;32m# Generate baselines \033[0m
+    > %s
+""" % ((os.path.basename(args[0]), ) * 4),
+        description=description,
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter
+    )
+
+    CIME.utils.setup_standard_logging_options(parser)
+
+    parser.add_argument("caseroot", nargs="?", default=os.getcwd(),
+                        help="Case directory")
+
+    parser.add_argument("-b", "--baseline-dir",
+                        help="Use custom baseline dir")
+
+    args = parser.parse_args(args[1:])
+
+    CIME.utils.handle_standard_logging_options(args)
+
+    return args.caseroot
+
+###############################################################################
+def _main_func(description):
+###############################################################################
+    caseroot, baseline_dir = parse_command_line(sys.argv, description)
```

Comment:
Tries to get caseroot and baseline_dir, but only caseroot is returned from the parse_command_line function